src/snapshots/let_polymorphism_lists.md

```diff
@@ -292,9 +292,12 @@ str_list = ["hello", "world"]
 float_list = [1.1, 2.2, 3.3]
 
 # Append empty list (polymorphic use)
-all_int_list = int_list + my_empty_list
-all_str_list = str_list + my_empty_list
-all_float_list = float_list + my_empty_list
+all_int_list = int_list + 
+my_empty_list
+all_str_list = str_list + 
+my_empty_list
+all_float_list = float_list + 
+my_empty_list
```

Comment:
this one also looks like a regression

Reply:
Same as above, this is caused by [this](https://github.com/JRI98/roc/blob/fe19ff4a44938df9f3efcb84ee6eacd0edfe8faa/src/snapshots/let_polymorphism_lists.md?plain=1#L235) malformed statement and is a separate issue where `++` is being transformed to `+` (and a malformed statement) in the formatted code.
For reference, this is the original line:
```
all_float_list = float_list ++ my_empty_list
```